### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by naamapps** [fujidaiti/smooth_sheets#126] [General]
I see you put your time and effort into making this package a great one (and it already is!).

This package takes my apps to another level of UX and fluidity.

So thank you for your hard work and dedication, I and many others appreciate your effort 🙏

So excited to see new features in the future. 

P.S. If you want to see what I've accomplished using your package, take a look at Mainspring - my habit tracker app.

iOS: https://apps.apple.com/app/id6467129951

Android: https://play.google.com/store/apps/details?id=com.naamapps.mainspring

**Comment by appinteractive** [comment]
I just want to do the same and thank you @fujidaiti for your dedication in making smooth_sheets the go to Package for sheets in Flutter. It's pretty rare that someone puts such emphasis on details into a package. 

I can also share an app of mine that we just released im beta a couple of days ago. It's an AI Travel Guide that I enriches your journey by being your audio guide wherever you are. 

It's available on [iOS](https://apps.apple.com/en/app/joaia/id6477830563) and [Android](https://play.google.com/store/apps/details?id=ch.holoai.joaia)

https://github.com/fujidaiti/smooth_sheets/assets/432828/a7a97d25-841c-49ff-8f57-df3301d092dd

**Reply by fujidaiti** [reply]
Wow, that's a nice hero animation! Is this a standard Hero widget?

**Reply by appinteractive** [reply]
Thanks for the compliment and yes that's a standard hero animation with `transitionOnUserGesture: true` so the magic don't break down while sliding down and a `placeholderBuilder`.

**Reply by fujidaiti** [reply]
>with transitionOnUserGesture: true so the magic don't break down while sliding down

I didn't know that. It can be a good example of the hero animation and the modal sheet.

**Comment by fujidaiti** [comment]
Hi, @naamapps and @appinteractive.

Thank you for using my package and such news motivate me a lot!
If you don't mind, could I list your apps in the README as real world examples?

**Reply by naamapps** [reply]
I would be honored to be featured on the readme.

**Reply by appinteractive** [reply]
Yes of cause! I would be honored too. 😌
